### PR TITLE
fix issue #2221

### LINKED ISSUE #2221: Korean texts are incorrectly shown in some tooltips
Some long tooltips are displayed wrong in korean. For instance,
correct one(english): This is a tooltip of long options that blah blah texts. You may translate it.
incorrect one(korean): This is a tooltip of (newline) blah texts. You may translate (newline) This can be changed by blah blah

Precisely speaking, not all tooltip texts are shown in the tooltip box in korean.
And some not related texts are also displayed together.

This issue occurs for some tooltips, not for all.

Sample image:
Tooltip text

### PR DESCRIPTION
**Pull request by kexplo**
It fixes #2221

### CODE REVIEW

File: src/windows/tooltip.c [hunk 1]
```diff
@@ -93,6 +93,7 @@ void window_tooltip_show(rct_string_id id, int x, int y)
 	RCT2_GLOBAL(0x0142006C, sint32) = -1;
 	char* buffer = RCT2_ADDRESS(RCT2_ADDRESS_COMMON_STRING_FORMAT_BUFFER, char);
 
+	memset(buffer, 0, 512);
```
Comment (IntelOrca) [inline src/windows/tooltip.c]: why is this necessary?

Comment (kexplo) [inline src/windows/tooltip.c]: Oh, I will remove this line.
Now, It is not necessary.

**Comment by kexplo** [comment]
How to retry AppVeyor build?

**Comment by kexplo** [comment]
Okay, I will
Thanks

**Pull request merged**